src/App.tsx

```diff
@@ -33,171 +73,95 @@ export const App: React.FC = () => {
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">
-              Todo is being saved now
-            </span>
-
-            <button type="button" className="todo__remove" data-cy="TodoDelete">
-              ×
-            </button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
+        {areTodosExist && <TodoList todos={filteredTodos} />}
         {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
-
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+        {areTodosExist && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {notCompletedTodosCount} items left
+            </span>
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={classNames('filter__link', {
+                  selected: selectedTodos === FilterTypes.All,
+                })}
+                data-cy="FilterLinkAll"
+                onClick={() => {
+                  if (selectedTodos !== FilterTypes.All) {
+                    setSelectedTodos(FilterTypes.All);
+                  }
+                }}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={classNames('filter__link', {
+                  selected: selectedTodos === FilterTypes.Active,
+                })}
+                data-cy="FilterLinkActive"
+                onClick={() => {
+                  if (selectedTodos !== FilterTypes.Active) {
+                    setSelectedTodos(FilterTypes.Active);
+                  }
+                }}
+              >
+                Active
+              </a>
```

Comment:
You can use your enum to render a list of these links. `Object.values/keys/entries` convert your enum into array